templates/run-benchmark.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+# vim: ft=sh
+set -e
+
+NIMBUS_ETH1_BENCHMARKS_REPO="/data/status-im/nimbus-eth1-benchmarks"
+NIMBUS_ETH1_REPO="/data/status-im/nimbus-eth1"
+NIMBUS_ETH1_TEMPLATE_DB="/data/template-db"
+NIMBUS_ETH1_DB_DIR="${NIMBUS_ETH1_REPO}/data/shared_mainnet_0"
+BENCHMARK_FILE_NAME='blocks-import-benchmark.csv'
+HOST_BENCHMARKS_DIR='/data/host-benchmarks'
+
+ERA1_DIR='/data/era1'
+ERA_DIR='/data/era'
+
+NIMBUS_ETH1_REPO_URL='https://github.com/status-im/nimbus-eth1'
+BENCHMARKS_REPO_URL="git@github.com:status-im/nimbus-eth1-benchmarks.git"
+BRANCH='master'
+
+FORCE_RUN="false"
+
+NIMBUS_ETH1_SERVICE_USER="nimbus"
+NIMBUS_ETH1_SERVICE_GROUP="staff"
```

Comment:
Use `$(id -u -n)` and `$(id -g -n)`, these variables are pointless, and also hardcoded.